mirrord/config/src/lib.rs

```diff
@@ -119,8 +118,9 @@ pub struct LayerConfig {
     #[config(nested)]
     pub feature: FeatureConfig,
 
-    #[config(nested, toggleable)]
-    pub operator: OperatorConfig,
+    /// Allow to lookup if operator is installed on cluster and use it
+    #[config(env = "MIRRORD_OPERATOR_ENABLE", default = true)]
```

Comment:
I'm not sure we want to allow non-operator connections in clusters with operators in them. Might be worth discussing in the channel.